common/transport/mqtt/src/main/java/org/thingsboard/server/transport/mqtt/MqttTransportHandler.java

```diff
@@ -395,11 +402,11 @@ private void handleSparkplugPublishMsg(ChannelHandlerContext ctx, String topicNa
                         break;
                     case NBIRTH:
```

Comment:
I propose to simplify this in the following way

```java
                    case NBIRTH:
                        sparkplugSessionHandler.setNodeBirthMetrics(sparkplugBProtoNode.getMetricsList());
                    case NCMD:
                    case NDATA:
                        sparkplugSessionHandler.onAttributesTelemetryProto(msgId, sparkplugBProtoNode, deviceName, sparkplugTopic);
                        break;
```

Reply:
Ok

Reply:
Please review my suggestion carefully. You have missed: ```sparkplugSessionHandler.setNodeBirthMetrics(sparkplugBProtoNode.getMetricsList());```